C++/C.sublime-syntax

```diff
@@ -884,6 +884,9 @@ contexts:
         - meta_scope: debug.type-declaration.union
         - include: union-keyword
         - include: or_pop
+    - match: (?={{identifier}};)
+      pop: true
+      comment: Used for typedefing things defined in types.
```

Comment:
Why do you choose to put this as an actual YAML key/value instead of a comment?

Reply:
I don't actually have a good reason for that :)